packages/pluggableWidgets/accordion-web/CHANGELOG.md

```diff
@@ -8,5 +8,6 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 -   Configure groups with a header & content.
```

Comment:
Isn't changelog here to allows users to see changes? This is a new widget, so I don't think is even necessary a changelog

Reply:
I think it adds value to add the added features on a high level. The [keep a changelog site](https://keepachangelog.com/en/1.0.0/) also suggests to do this in the example.